app.js

```diff
@@ -0,0 +1,217 @@
+    if (usrInput.type === "radio") {
+      usrInput.checked = false;
+    } else {
+      usrInput.value = "";
+    }
+  }
+};
+
+const cancelContactModal = () => {
+  toggleBackdrop();
+  closecontactModal();
+  clearUsrInputs();
+};
+
+const updateUi = () => {
+  if (contacts.length == "") {
+    enteryText.style.display = "block";
+  } else {
+    enteryText.style.display = "none";
+  }
+};
+
+const cancelContactDeletion = () => {
+  toggleBackdrop();
+  deleteModal.classList.remove("visible");
+  updateUi();
+};
+
+const deleteContactElement = (contactId) => {
+  deleteModal.classList.add("visible");
+  toggleBackdrop();
+  successBtn.replaceWith(successBtn.cloneNode(true));
+  successBtn = document.getElementById("btn-success");
+  dangerBtn.addEventListener("click", cancelDeletionModal);
+  successBtn.addEventListener(
+    "click",
+    deleteContactConfirmation.bind(null, contactId)
+  );
+};
+
+const deleteContactConfirmation = (contactId) => {
+  let contactIndex = 0;
+  for (const contact of contacts) {
+    if (contact.id === contactId) {
+      break;
+    }
+    contactIndex++;
+  }
+  contacts.splice(contactIndex, 1);
+  const newContactList = document.getElementById("contact-list");
+  newContactList.children[contactIndex].remove();
+  cancelContactDeletion();
+  updateUi();
+};
+//------------------------  Edit Contact ----------------------//
+
+const editContactHandler = (contId) => {
+  const selectedContact = contacts.filter((el) => el.id === contId);
+  console.log(selectedContact);
+  editContactModal.classList.add("visible");
+  toggleBackdrop();
+};
+
+//------------------------  Edit Contact ----------------------//
+
+const renderContactElement = (
+  id,
+  firstName,
+  lastName,
+  imageUrl,
+  emailAdd,
+  phoneNo,
+  status
+) => {
+  const editId = `edit${Math.random()}`;
+  const deleteId = `delete${Math.random()}`;
+
+  const newContactElement = document.createElement("div");
+  newContactElement.className = "contact-element";
+  newContactElement.innerHTML = `
+       <div class="contact-element__image">
+       <img src="${imageUrl}" alt="${firstName}">
+       </div>
+       <div class="contact-element__info">
+       <h2>${firstName} ${lastName}</h2>
+       <h4 style= "color:${
+         status === "Active" ? "green" : "red"
+       }">${status}</h4>
+       <p>${emailAdd}</p>
+       <p>${phoneNo}</p>
+       <div>
+       <button class="btn btn--success nasir" id=${editId}>Edit</button>
+       <button class="btn btn--danger" id=${deleteId}>Delete</button>
+       </div>
+       </div>
+      `;
+  const newContactlist = document.getElementById("contact-list");
+  newContactlist.append(newContactElement);
+  const deleteBtn = document.getElementById(deleteId);
+  deleteBtn.addEventListener("click", deleteContactElement.bind(null, id));
+  const editBtn = document.getElementById(editId);
+  editBtn.addEventListener("click", editContactHandler);
+};
+
+const closecontactModal = () => {
```

Comment:
can we use the naming convention properly? In camel case it should be `closeContactModal`